src/web/mjs/connectors/mangaokutr.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class mangatrnet extends WordPressMadara {
```

Comment:
I recommend to match the class name with the filename (using pascal case for class name)